README.md

```diff
@@ -33,30 +33,74 @@ Prerequisites
     $ cd libtock-c
     ```
 
-1. The main requirement to build the C applications in this repository
-   is a cross compiler for embedded targets. Currently, the only
-   supportetd target is ARM Cortex M, so you will need an
-   `arm-none-eabi` toolchain.
+1. The main requirement to build the C applications in this repository is having
+   cross compilers for embedded targets. You will need an `arm-none-eabi`
+   toolchain for Cortex-M targets, and `riscv64-unknown-elf` toolchain for
+   RISC-V targets.
 
-   MacOS:
+   **MacOS**:
    ```
    $ brew tap ARMmbed/homebrew-formulae && brew update && brew install arm-none-eabi-gcc
+   $ brew tap riscv/riscv && brew update && brew install riscv-gnu-toolchain --with-multilib
```

Comment:
How important is `--with-multilib` / what is it doing? That causes brew to install from source, which fails on my machine:

(relevant bits of the logs):

```
checking for ANSI C header files... yes
checking for the correct version of mpc.h... rm: conftest.dSYM: is a directory
no
configure: error: Building GCC requires GMP 4.2+, MPFR 3.1.0+ and MPC 0.8.0+.
Try the --with-gmp, --with-mpfr and/or --with-mpc options to specify
their locations.  Source code for these libraries can be found at
their respective hosting sites as well as at
https://gcc.gnu.org/pub/gcc/infrastructure/.  See also
http://gcc.gnu.org/install/prerequisites.html for additional info.  If
you obtained GMP, MPFR and/or MPC from a vendor distribution package,
make sure that you have installed both the libraries and the header
files.  They may be located in separate packages.
rm: conftest.dSYM: is a directory
make: *** [stamps/build-gcc-newlib-stage1] Error 1
make: *** Waiting for unfinished jobs....
```

It's *probably* a missing dependency in the brew formula?

Reply:
Very important, it gets you a 32 bit compiler (in addition to the 64 bit standard version).

Reply:
As a future reference point, if people are having issues, the inexplicable `brew reinstall gmp mpfr libmpc` fixed this for me (really just `libmpc` in this case, but let's be thorough).

Yes, it was already installed. Yes, `brew info libmpc` confirmed that. No, `brew doctor` reported no errors. Yes, brew was confused that I wanted to reinstall. Yes, that somehow fixed the problem. Yes, I'm mildly+ annoyed.